connect/runtime/src/test/java/org/apache/kafka/connect/storage/FileOffsetBackingStoreTest.java

```diff
@@ -57,9 +66,12 @@ public class FileOffsetBackingStoreTest {
 
     @Before
     public void setup() throws IOException {
-        store = new FileOffsetBackingStore();
+        converter = mock(Converter.class);
+        when(converter.toConnectData(anyString(), any(byte[].class))).thenReturn(new SchemaAndValue(null,
```

Comment:
A comment here could be helpful:
```suggestion
        // This is only needed for storing deserialized connector partitions, which we don't test in most of the cases here
        when(converter.toConnectData(anyString(), any(byte[].class))).thenReturn(new SchemaAndValue(null,
```